pkgs/applications/virtualization/charliecloud/test.sh

```diff
@@ -0,0 +1,42 @@
+#/bin/env bash
```

Comment:
Having a test is nice, but it is probably not so useful when they are not automatically run. Such tests tend to get stale over time.

What you could do after this PR (if you are interested in putting in the time) is write a NixOS test. If you are not familiar with NixOS tests, they run in a VM and therefore provide the opportunity to set up a real testing environment. E.g., you could automatically set up a Docker registry in the test (there is already a NixOS test that does this, that can be used for inspiration, `nixos/tests/docker-registry.nix`) and then test charliecloud against that registry.

Reply:
Okay! I wrote the test. I used `dockerTools.examples.nix` as a base image. Hope this is fine.

Reply:
Great! I amended the commit, seems that you forgot to commit the change to `alltests.nix`. So, should be good now.

Let's see if ofborg is also happy.

Reply:
Oh, yes sorry. Ofborg has passed the `passthru.tests.charliecloud` test anyway, I checked that

Reply:
It didn't, it just skips that step if `passthru.tests` cannot be properly be evaluated.